src/serverlist.cpp

```diff
@@ -234,7 +234,10 @@ CServerListManager::CServerListManager ( const quint16  iNPortNum,
     }
     else
     {
-        qWarning() << "Ignoring invalid serverinfo, please verify the parameter syntax.";
+        if ( !strServerInfo.isEmpty() )
```

Comment:
Why is this repeating the check done on line 175?  This `else` block is only entered `if ( !strServerInfo.isEmpty() )`.

Reply:
Then the issue is somewhere else probably.